QMake: undefined symbol: __wasm_lpad_context

It seems like the QMake builds have been accidently broken. Need to figure out how to fix it.

Build extract from a `build_qmake.bat` run:
```
em++: warning: USE_PTHREADS + ALLOW_MEMORY_GROWTH may run non-wasm code slowly, see https://github.com/WebAssembly/design/issues/1271 [-Wpthreads-mem-growth]
wasm-ld: error: main.o: undefined symbol: __wasm_lpad_context
wasm-ld: error: main.o: undefined symbol: __wasm_lpad_context
wasm-ld: error: main.o: undefined symbol: __wasm_lpad_context
wasm-ld: error: main.o: undefined symbol: __wasm_lpad_context
wasm-ld: error: main.o: undefined symbol: __wasm_lpad_context
wasm-ld: error: main.o: undefined symbol: __wasm_lpad_context
wasm-ld: error: main.o: undefined symbol: __wasm_lpad_context
wasm-ld: error: main.o: undefined symbol: __wasm_lpad_context
em++: error: '/emsdk/upstream/bin/wasm-ld -o ./helloworld.wasm /opt/Qt/lib/libQt6BundledFreetype.a /opt/Qt/lib/libQt6BundledLibpng.a main.o helloworld.js_plugin_import.o helloworld.js_qml_plugin_import.o qrc_qml.o /opt/Qt/lib/objects-Release/QWasmIntegrationPlugin_resources_1/.rcc/qrc_wasmfonts.cpp.o /opt/Qt/lib/objects-Release/QWasmIntegrationPlugin_resources_2/.rcc/qrc_wasmwindow.cpp.o /opt/Qt/lib/objects-Release/Gui_resources_1/.rcc/qrc_qpdf.cpp.o /opt/Qt/lib/objects-Release/Gui_resources_2/.rcc/qrc_gui_shaders.cpp.o /opt/Qt/plugins/platforms/libqwasm.a /opt/Qt/plugins/iconengines/libqsvgicon.a /opt/Qt/plugins/imageformats/libqgif.a /opt/Qt/plugins/imageformats/libqicns.a /opt/Qt/plugins/imageformats/libqico.a /opt/Qt/plugins/imageformats/libqjpeg.a /opt/Qt/lib/libQt6BundledLibjpeg.a /opt/Qt/plugins/imageformats/libqsvg.a /opt/Qt/lib/libQt6Svg.a /opt/Qt/plugins/imageformats/libqtga.a /opt/Qt/plugins/imageformats/libqtiff.a /opt/Qt/plugins/imageformats/libqwbmp.a /opt/Qt/plugins/imageformats/libqwebp.a /opt/Qt/./qml/QtQml/Base/objects-Release/Qml_resources_1/.rcc/qrc_qmake_QtQml_Base.cpp.o /opt/Qt/lib/objects-Release/Qml_resources_2/.rcc/qrc_qmlMetaQmldir.cpp.o /opt/Qt/plugins/qmltooling/libqmldbg_debugger.a /opt/Qt/./qml/QtQuick/objects-Release/Quick_resources_1/.rcc/qrc_qmake_QtQuick.cpp.o /opt/Qt/lib/objects-Release/Quick_resources_2/.rcc/qrc_scenegraph_shaders.cpp.o /opt/Qt/./qml/QtQml/Models/objects-Release/QmlModels_resources_1/.rcc/qrc_qmake_QtQml_Models.cpp.o /opt/Qt/plugins/qmltooling/libqmldbg_inspector.a /opt/Qt/plugins/qmltooling/libqmldbg_local.a /opt/Qt/plugins/qmltooling/libqmldbg_messages.a /opt/Qt/plugins/qmltooling/libqmldbg_native.a /opt/Qt/plugins/qmltooling/libqmldbg_nativedebugger.a /opt/Qt/plugins/qmltooling/libqmldbg_preview.a /opt/Qt/plugins/qmltooling/libqmldbg_profiler.a /opt/Qt/plugins/qmltooling/libqmldbg_quick3dprofiler.a /opt/Qt/lib/libQt6Quick3DUtils.a /opt/Qt/plugins/qmltooling/libqmldbg_quickprofiler.a /opt/Qt/plugins/qmltooling/libqmldbg_server.a /opt/Qt/lib/libQt6PacketProtocol.a /opt/Qt/plugins/qmltooling/libqmldbg_tcp.a /opt/Qt/plugins/tls/libqcertonlybackend.a /opt/Qt/qml/QtQuick/libqtquick2plugin.a /opt/Qt/lib/libQt6Quick.a /opt/Qt/lib/libQt6OpenGL.a /opt/Qt/lib/libQt6Gui.a /opt/Qt/lib/libQt6BundledHarfbuzz.a /opt/Qt/lib/libQt6BundledFreetype.a /opt/Qt/lib/libQt6BundledLibpng.a /opt/Qt/./qml/QtQml/objects-Release/QmlMeta_resources_1/.rcc/qrc_qmake_QtQml.cpp.o /opt/Qt/qml/QtQml/libqmlmetaplugin.a /opt/Qt/qml/QtQml/Base/libqmlplugin.a /opt/Qt/qml/QtQml/Models/libmodelsplugin.a /opt/Qt/lib/libQt6QmlModels.a /opt/Qt/./qml/QtQml/WorkerScript/objects-Release/QmlWorkerScript_resources_1/.rcc/qrc_qmake_QtQml_WorkerScript.cpp.o /opt/Qt/qml/QtQml/WorkerScript/libworkerscriptplugin.a /opt/Qt/lib/libQt6QmlWorkerScript.a /opt/Qt/./qml/QtQuick/Window/objects-Release/quickwindow_resources_1/.rcc/qrc_qmake_QtQuick_Window.cpp.o /opt/Qt/qml/QtQuick/Window/libquickwindowplugin.a /opt/Qt/lib/libQt6Qml.a /opt/Qt/lib/libQt6Network.a /opt/Qt/lib/libQt6Core.a /opt/Qt/lib/libQt6BundledZLIB.a /opt/Qt/lib/libQt6BundledPcre2.a -L/emsdk/upstream/emscripten/cache/sysroot/lib/wasm32-emscripten /emsdk/upstream/emscripten/cache/sysroot/lib/wasm32-emscripten/crtbegin.o --whole-archive -lfetch-mt -lembind-rtti --no-whole-archive -lGL-mt-webgl2 -lal -lhtml5 -lstubs -lnoexit -lc-mt -ldlmalloc-mt -lcompiler_rt-mt -lc++-mt-noexcept -lc++abi-mt-noexcept -lsockets-mt -mllvm -combiner-global-alias-analysis=false -mllvm -enable-emscripten-sjlj -mllvm -disable-lsr --import-undefined --import-memory --shared-memory --strip-debug --export-if-defined=main --export-if-defined=_emscripten_thread_init --export-if-defined=_emscripten_thread_exit --export-if-defined=_emscripten_thread_crashed --export-if-defined=_emscripten_tls_init --export-if-defined=pthread_self --export-if-defined=__start_em_asm --export-if-defined=__stop_em_asm --export-if-defined=__start_em_lib_deps --export-if-defined=__stop_em_lib_deps --export-if-defined=__start_em_js --export-if-defined=__stop_em_js --export-if-defined=__main_argc_argv --export=stackSave --export=stackRestore --export=stackAlloc --export=__wasm_call_ctors --export=__errno_location --export=emscripten_dispatch_to_thread_ --export=_emscripten_thread_free_data --export=emscripten_main_browser_thread_id --export=emscripten_main_thread_process_queued_calls --export=emscripten_run_in_main_runtime_thread_js --export=emscripten_stack_set_limits --export=malloc --export=free --export=saveSetjmp --export=setThrew --export=emscripten_builtin_memalign --export=__cxa_is_pointer_type --export=ntohs --export=htons --export=__dl_seterr --export=htonl --export-table -z stack-size=65536 --initial-memory=52428800 --no-entry --max-memory=2147483648 --global-base=1024' failed (returned 1)
make: *** [Makefile:347: helloworld.js] Error 1
```

--- a/sample/myproject.pro
+++ b/sample/myproject.pro
@@ -11,6 +11,7 @@ INCLUDEPATH  += /opt/wasm-deps/include/eigen3
 
 # enable C++ exception catching
 QMAKE_CXXFLAGS += -fwasm-exceptions
+QMAKE_LFLAGS += -fwasm-exceptions
 
 # embed example.txt into binary
 QMAKE_PRE_LINK += $$QMAKE_COPY ../source/example.txt .

